tensorflow/compiler/mlir/hlo/lib/CAPI/CMakeLists.txt

```diff
@@ -3,8 +3,16 @@ add_mlir_public_c_api_library(MLIRHLOCAPIDialects
   Dialects.cpp
   Types.cpp
   Passes.cpp
+
   LINK_LIBS PUBLIC
   ChloDialect
   MhloDialect
+  ChloPasses
+  MhloPasses
+  MhloToLhloConversion
+  MhloToArithmeticConversion
+  MhloToMemrefConversion
+  MhloToStandard
+  MhloToLinalg
+  MhloShapeOpsToStandard
 )
-target_link_libraries(MLIRHLOCAPIDialects PRIVATE AllMhLoPasses)
```

Comment:
I don't think copy and pasting the passes defined by this AllMhloPasses target is the right approach.

It is defined as


add_library(AllMhloPasses INTERFACE)
target_link_libraries(AllMhloPasses INTERFACE
...

so I am curious why this build no longer works?

Reply:
Because `AllMhloPasses` is not defined by `add_mlir_library` which is import for MLIR's python bindling library. Also it's related with how to integrate the python binding. It's hard for me to create a standalone to show how we integrated the python binding.   
But I think firstly it's necessary to fix the spelling mistake.

Reply:
Yeah, I have a CL for that.